### DISCUSSION TITLE
How Do I Make A Tag Page For A Tag With an Apostrophe?

### DISCUSSION THREAD

**Discussion by Thysbelon** [11ty/eleventy#3202] [Q&A] [answered]
I'm automatically generating tag pages for all of my tags. One of my tags contains an apostrophe. The URL generated is 
`/Tags/example%27s-tag/` for the tag "example's tag". This URL is fine, *however*, the name of the folder for the tag generated in the `_site` folder is also `example%27s-tag`; this folder's index.html is impossible to navigate to unless I type 
`/Tags/example%2527s-tag/` in the browser's address bar.
I tested this with a simple test server: if I manually change the name of the folder to `example's-tag`, it seems to work correctly.
I would like to know how to configure Eleventy to set the name of the folder to `example's-tag`.
Thank you for reading.

**Comment by Thysbelon** [comment, marked as answer]
I think nunjucks is the reason I was struggling. I made two similar templates:

`test's.njk`
```
---
customKey: "test's"
permalink: "/{{ customKey | replace('&#39;',`'`) }}/"
---
<h1>Test</h1>
```
`test's2.11ty.js`
```javascript
module.exports = {

data() {
	return {
		customKey: "test's2",
		permalink: data => `/${data.customKey}/`
	}
},

render(data) {
return `
	<h1>test2</h1>
`;
}
}

```

The njk template generated `test&#39;s`. The 11ty.js template generated `test's2`

**Comment by pdehaan** [comment]
I think you'd want to use the [`slugify` filter](https://www.11ty.dev/docs/filters/slugify/) on the `customKey` permalink.

**Reply by pdehaan** [reply]
```njk
---
customKey: "test's"
permalink: "/{{ customKey | slugify }}/"
---
<h1>{{ customKey }}</h1>
```

```sh
npm run build 


> 11ty-3202@1.0.0 build
> eleventy

[11ty] Writing www/test-s/index.html from ./src/test's.njk
[11ty] Wrote 1 file in 0.03 seconds (v2.0.1)
```
